### PR TITLE
Azure Instances Log Analytics Utilization

### PR DESCRIPTION
**Pull request by srpomeroy**
### Description

This policy captures CPU and Memory utilization metrics from Azure Log Analytics and compares those values to the CPU and Memory utilization parameters which define the thresholds to raise an incident for.

The escalation will tag all identified instances to later be resized by the Downsize Instances policy.

### Issues Resolved

N/A

### Contribution Check List

- [X] All tests pass.
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Comment by cdwilhelm** [comment]
@srpomeroy  also need to update the repo root README with the new policy

File: cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+
+parameter "param_take_action_tag" do
+  category "User Inputs"
+  label "Downsize Tag Key:Value"
+  description "Cloud native instance tag name and value that will be added to an instance identified for action"
+  type "string"
+  default "downsize:true"
+end
+
+parameter "param_email" do
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt]: Need to shorten the label and add description.  
```
parameter "param_email" do
  type "list"
  label "Email addresses"
  description "Email addresses of the recipients you wish to notify"
end```

File: cost/azure/instances_log_analitics_utilization/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+## Azure Instances Utilization from Log Analytics
+
+### What it does
+
+This Policy Template uses monitoring metrics from Log Analytics for the last 30 days to identify underutilized instances. This is meant to be run as a monthly policy.
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_utilization/README.md]: Monthly is not an option in the frequency.

File: cost/azure/instances_log_analitics_utilization/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+## Azure Instances Utilization from Log Analytics
+
+### What it does
+
+This Policy Template uses performance metrics from Log Analytics from the last 30 days to identify underutilized instances. This is meant to be run as a monthly policy. 
+
+### Prerequisites
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_utilization/README.md]: Does the vm need the logging agent?

File: cost/azure/instances_log_analitics_utilization/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+## Azure Instances Utilization from Log Analytics
+
+### What it does
+
+This Policy Template is used to collect monitoring metrics for VMs from Log Analytics to identify underutilized instances.
+
+### Prerequisites
+
+- Azure Service Principal (AKA Azure Active Directory Application) with the appropriate permissions to manage resources in the target subscription
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply cost/azure/instances_log_analitics_utilization/README.md]: Yep, fixed

File: cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt (hunk 1 above)
Comment (srpomeroy) [reply cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt]: Done.

File: cost/azure/instances_log_analitics_utilization/README.md (hunk 3 above)
Comment (srpomeroy) [reply cost/azure/instances_log_analitics_utilization/README.md]: Yep. Fixed

File: cost/azure/instances_log_analitics_utilization/README.md (hunk 2 above)
Comment (srpomeroy) [reply cost/azure/instances_log_analitics_utilization/README.md]: Revised to weekly until monthly is available.

File: cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt [hunk 5]
```diff
@@ -0,0 +1,522 @@
+name "Azure Instances Utilization from Log Analytics"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers instance utilization data from Azure Log Analytics and tags underutilized instances.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Cost"
+
```
Comment (cdwilhelm) [inline cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt]: Need to add the permissions declaration for the cred

Comment (srpomeroy) [reply cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt]: Added

**Pull request merged**